lighthouse-cli/run.ts

```diff
@@ -36,7 +36,7 @@ function formatArg(arg: string, value: any): string {
   }
 
   // Only quote the value if it contains spaces
-  if (value.indexOf(' ') !== -1) {
+  if (typeof(value) === 'string' && value.indexOf(' ') !== -1) {
```

Comment:
not sure but maybe we should do `String(value)` but i'll leave that decision to @brendankenny @paulirish @patrickhulce

Reply:
Regarding the first point, I have always felt that being explicit is better for `typeof`. For example `typeof val === true` has implied parens, but if you didn't know better, I suppose you could be forgiven for expecting them to be `typeof(val === true)`. (poor example, but hopefully it demonstrates the point)

Supper happy to yield on this, just my 2 cents!

Reply:
In lighthouse we use `typeof var` everywhere without parens because it's an operator not a function. Nonetheless it's just codestyle and I'm all for consistency.